cms/types/generated/contentTypes.d.ts

```diff
@@ -1199,6 +1199,7 @@ export interface ApiOrganizationThemeOrganizationTheme
   };
   attributes: {
     name: Attribute.String & Attribute.Required & Attribute.Unique;
+    order: Attribute.Integer & Attribute.DefaultTo<1>;
```

Comment:
According to the rest of the code (especially `cms/src/api/organization-theme/content-types/organization-theme/schema.json`), the default should be 0, no?

Reply:
I finally changed it manually as I couldn't get it generated from the CMS interaction or exporting the config. I'm not sure if this actually affects the CMS behavior, though.